pkg/controller/deployment/util/deployment_util.go

```diff
@@ -536,7 +536,7 @@ func GetNewReplicaSet(deployment *extensions.Deployment, c clientset.Interface)
 // RsListFromClient returns an rsListFunc that wraps the given client.
 func RsListFromClient(c clientset.Interface) RsListFunc {
 	return func(namespace string, options metav1.ListOptions) ([]*extensions.ReplicaSet, error) {
-		rsList, err := c.Extensions().ReplicaSets(namespace).List(options)
+		rsList, err := c.ExtensionsV1beta1().ReplicaSets(namespace).List(options)
```

Comment:
Yes. I'd like to encourage people to explicitly spell out the version in their code, rather than get implicitly bumped to a new API version when imports a newer version of the clients.